### PR TITLE
PUBDEV-7352 - Enable import of Stacked Ensemble MOJO models

### PR DESCRIPTION
**Pull request by Pscheidl**
https://0xdata.atlassian.net/browse/PUBDEV-7352

Scoring only. No emphasis put on model info and output. Yet the output seems to ok.

- Java test verifies MOJO importability/MOJO download + scoring correctness + byte-wise mojo comparison.
- Python & R tests verify mojo import/download + scoring functionality. No comparison of output is done to the original model. Aim of this PR is only to provide scoring.

<details><summary>Original model output</summary>

```
Model Details:
==============

H2OBinomialModel: stackedensemble
Model ID:  StackedEnsemble_model_R_1584442967663_1407 
Number of Base Models: 2

Base Models (count by algorithm type):

drf gbm 
  1   1 

Metalearner:

Metalearner algorithm: glm


H2OBinomialMetrics: stackedensemble
** Reported on training data. **

MSE:  0.1420669
RMSE:  0.3769177
LogLoss:  0.4517939
Mean Per-Class Error:  0.1910265
AUC:  0.8993468
AUCPR:  0.8864197
Gini:  0.7986936

Confusion Matrix (vertical: actual; across: predicted) for F1-optimal threshold:
          0    1    Error       Rate
0      1651  679 0.291416  =679/2330
1       242 2428 0.090637  =242/2670
Totals 1893 3107 0.184200  =921/5000

Maximum Metrics: Maximum metrics at their respective thresholds
                        metric threshold       value idx
1                       max f1  0.441837    0.840575 229
2                       max f2  0.336658    0.903414 280
3                 max f0point5  0.625085    0.837175 136
4                 max accuracy  0.468397    0.819600 216
5                max precision  0.895850    1.000000   0
6                   max recall  0.145604    1.000000 372
7              max specificity  0.895850    1.000000   0
8             max absolute_mcc  0.466264    0.639403 217
9   max min_per_class_accuracy  0.526935    0.812017 186
10 max mean_per_class_accuracy  0.502027    0.815816 198
11                     max tns  0.895850 2330.000000   0
12                     max fns  0.895850 2615.000000   0
13                     max fps  0.069763 2330.000000 399
14                     max tps  0.145604 2670.000000 372
15                     max tnr  0.895850    1.000000   0
16                     max fnr  0.895850    0.979401   0
17                     max fpr  0.069763    1.000000 399
18                     max tpr  0.145604    1.000000 372

Gains/Lift Table: Extract with `h2o.gainsLift(<model>, <data>)` or `h2o.gainsLift(<model>, valid=<T/F>, xval=<T/F>)`
```

</details>

<details><summary>Generic output</summary>

```
Model Details:
==============

H2OBinomialModel: generic
Model ID:  Generic_model_R_1584442967663_1410 
NULL


H2OBinomialMetrics: generic
** Reported on training data. **

MSE:  0.1420669
RMSE:  0.3769177
LogLoss:  0.4517939
Mean Per-Class Error:  0.1910265
AUC:  0.8993468
AUCPR:  0.8864197
Gini:  0.7986936
R^2:  0.4290924

Confusion Matrix (vertical: actual; across: predicted) for F1-optimal threshold:
          0    1    Error       Rate
0      1651  679 0.291416  =679/2330
1       242 2428 0.090637  =242/2670
Totals 1893 3107 0.184200  =921/5000

Maximum Metrics: Maximum metrics at their respective thresholds
                        metric threshold       value idx
1                       max f1  0.441837    0.840575 229
2                       max f2  0.336658    0.903414 280
3                 max f0point5  0.625085    0.837175 136
4                 max accuracy  0.468397    0.819600 216
5                max precision  0.895850    1.000000   0
6                   max recall  0.145604    1.000000 372
7              max specificity  0.895850    1.000000   0
8             max absolute_mcc  0.466264    0.639403 217
9   max min_per_class_accuracy  0.526935    0.812017 186
10 max mean_per_class_accuracy  0.502027    0.815816 198
11                     max tns  0.895850 2330.000000   0
12                     max fns  0.895850 2615.000000   0
13                     max fps  0.069763 2330.000000 399
14                     max tps  0.145604 2670.000000 372
15                     max tnr  0.895850    1.000000   0
16                     max fnr  0.895850    0.979401   0
17                     max fpr  0.069763    1.000000 399
18                     max tpr  0.145604    1.000000 372

Gains/Lift Table: Extract with `h2o.gainsLift(<model>, <data>)` or `h2o.gainsLift(<model>, valid=<T/F>, xval=<T/F>)`
```

</details>

### CODE REVIEW

File: h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_stackedensemble.R [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    print(stackedensemble)
+    mojo_original_name <- h2o.download_mojo(model = stackedensemble, path = tempdir())
+    mojo_original_path <- paste0(tempdir(),"/",mojo_original_name)
+    
+    generic_model <- h2o.genericModel(mojo_original_path)
+    print(generic_model)
+    
+    generic_model_preds  <- h2o.predict(generic_model, train)
+    expect_equal(length(generic_model_preds), 3)
+    expect_equal(h2o.nrow(generic_model_preds), 5000)
```
Comment (michalkurka) [inline h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_stackedensemble.R]: do we check the predictions anywhere?

Comment (Pscheidl) [reply h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_stackedensemble.R]: JUnit: https://github.com/h2oai/h2o-3/pull/4416/files#diff-87f75a070de23cd6342c51d04c308a6bR905

Comment (Pscheidl) [reply h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_stackedensemble.R]: Also, check here please: https://github.com/h2oai/h2o-3/pull/4416/files#diff-87f75a070de23cd6342c51d04c308a6bR908

**Comment by Pscheidl** [comment]
Waiting for tests.

**Comment by Pscheidl** [comment]
DO NOT MERGE. Moved to 3.30.0.1 - base change required.

**Pull request merged**